### ISSUE TITLE
Docker Master: CSV bills import (cospend compatible) not working

### ISSUE THREAD

**Issue by AlexanderSch90** [spiral-project/ihatemoney#980]
support for cospend has been merged in master recently (CSV bills import (cospend compatible) #951) 
If I select the CSV export from Cospent, the error is: Unsupported file type.
i have tried several csv files. None of them worked. I can upload an example if needed if the issue is not self-reproducible.

**Comment by youegraillot** [comment]
Would you be kind enough to share your csv so I can debug this later ?

**Comment by AlexanderSch90** [comment]
yes no Problem
[spotify_2022-01-13.csv](https://github.com/spiral-project/ihatemoney/files/7871453/spotify_2022-01-13.csv)

**Comment by qoheniac** [comment]
If I try to import a CSV exported from Cospend, it fails telling me it's "Unable to parse CSV".

**Comment by almet** [comment]
Yes, this is not supported for now. Sorry for the inconvenience.

**Comment by almet** [comment]
Hi @qoheniac and @AlexanderSch90, U just found some time to work on this. It seems that the exported files from Cospend changed their format in 1.3.8 (see https://github.com/julien-nc/cospend-nc/blob/master/CHANGELOG.md#1318--2021-09-14)

From what I see from the file @AlexanderSch90 sent me, it seems that we have two different information in the same csv file, and our import mechanism needs to be updated.

**Comment by almet** [comment]
New format is not documented, but [the code](https://github.com/julien-nc/cospend-nc/blob/adf0c6fe7a957c19e4f47e4af48ea5d56de8e429/lib/Service/ProjectService.php#L5258-L5402) might help.

**Comment by almet** [comment]
So it exports different information in the same file : 

1. members: `name,weight,active,color`
2. bills: `what,amount,date,timestamp,payer_name,payer_weight,payer_active,owers,repeat,repeatfreq,repeatallactive,repeatuntil,categoryid,paymentmode,paymentmodeid,comment`
3. categories: `categoryname,categoryid,icon,color`
4. payment modes `paymentmodename,paymentmodeid,icon,color`
5. currencies `currencyname,exchange_rate`

**Comment by mihir1739** [comment]
Can I work on this?

**Comment by almet** [comment]
> Can I work on this?

Hi, thanks for proposing your help. In general open source projects work this way : you work on your side until you propose something to review to the project. At least, that's how we handle contributions here. So : don't hesitate to work on this.

**Comment by Travis544** [comment]
Hi, I will began working on this on my side.
